rest_framework/authtoken/management/commands/drf_create_token.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from rest_framework.authtoken.models import Token
+
+
+class Command(BaseCommand):
+    help = 'Create DRF Token for a given user'
+
+    def create_user_token(self, username):
+        user = User.objects.get(username=username)
```

Comment:
I would like to handle the situation when `username` doesn't exist. Should I let the code raise an exception here and try/except it in the handle() method (maybe displaying the error `"ERROR: given username does not exists"`) or is there a better way to do this?